weave-js/src/components/Panel2/panellib/libpanel.ts

```diff
@@ -65,8 +74,13 @@ export interface PanelSpec<X, C, T extends Type> {
   ) => Promise<C> | C;
   ConfigComponent?: React.ComponentType<PanelPropsInternal<any, C, X>>;
   Component: React.ComponentType<PanelPropsInternal<any, C, X>>;
-
   inputType: T;
+  /**
+   * UI rules are declarative way to toggle behavior in Weave panels
+   *
+   * Downstream code with configuration options will
```

Comment:
looks like uncompleted comment. Given the generic nature of the options allowed, I'm not sure what would be useful information to put here.